src/app/app.routing.ts

```diff
@@ -1,17 +1,22 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 import {ErrorComponent} from '../util/error/error.component';
-import {ModuleComponent} from "./module-component/module.component";
-import {SkillMatrixComponent} from "./skillmatrix-component/skillmatrix.component";
+import {ModuleComponent} from "./components/module-component/module.component";
+import {SkillMatrixComponent} from "./components/skillmatrix-component/skillmatrix.component";
+import {ModuleOverviewComponent} from "./components/module-overview-component/module-overview.component";
 
 const routes: Routes = [
   {
+    path: 'moduleoverview',
+    component: ModuleOverviewComponent
+  },
+  {
     path: 'module',
     component: ModuleComponent
   },
   {
     path: '',
```

Comment:
Doesn't work after a curriculum is selected nothing happens!

Reply:
I didn't change anything on the moduleoverview component part. This doesn't contribute to it not working.